modules/tooltip/css/lib/tooltip.scss

```diff
@@ -1,12 +1,10 @@
 .wdc-tooltip {
   @include wdc-type();
   @include wdc-type-body();
-  @include wdc-depth-3();
   display: inline-block;
   background-color: $_wdc-tooltip-bg-color;
-  padding: $wdc-spacing-s;
-  border-radius: 3px;
-  border: 1px solid $_wdc-tooltip-border-color;
+  padding: 0;
```

Comment:
Duplicate padding

Reply:
What do you mean?

Reply:
[Line 47](https://github.com/Workday/canvas-kit/pull/49/files#diff-e517748a8012b36b2b0623e4b4bb0444R47) overrides this. When you move those styles up, you'll see it haha.

Reply:
Oh I see